src/libpriv/rpmostree-postprocess.c

```diff
@@ -1485,7 +1485,11 @@ rpmostree_treefile_postprocessing (int            rootfs_fd,
                                                                 error))
       goto out;
 
-    if (base_version != NULL)
+    if (base_version != NULL && next_version == NULL)
```

Comment:
If `base_version` is `NULL`, then there is no `mutate-os-release` in the manifest, right?